colors/termschool.vim

```diff
@@ -127,3 +127,5 @@ hi SpellRare ctermfg=0 ctermbg=225 guifg=#000000 guibg=#ffdfff
 hi TabLineFill ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
 hi TabLineSel ctermfg=NONE ctermbg=59 cterm=NONE guifg=NONE guibg=#3f4b52 gui=NONE
 hi TabLine ctermfg=NONE ctermbg=233 cterm=NONE guifg=NONE guibg=#121212 gui=NONE
+hi SignColumn     ctermbg=NONE guibg=#2e373b
```

Comment:
Well, I don't see why the `guibg` is a problem since `guibg` and `guifg` are used when 24-bit colors are enabled with `termguicolors`, so `#2e373b` and generally anything between `#000000` and `#ffffff` is valid. Please let me know if I'm wrong here, as I said, I'm new to this.

In any case though, it doesn't matter cause I think it's better to have `SignColumn` and `FoldColumn` match `LineNr`.